xb.bat

```diff
@@ -29,12 +29,27 @@ REM Utilities
 REM ============================================================================
 
 :check_python
-SETLOCAL
+setlocal enabledelayedexpansion
 SET FOUND_PYTHON_EXE=""
-1>NUL 2>NUL CMD /c where python3
-IF NOT ERRORLEVEL 1 (
-  SET FOUND_PYTHON_EXE=python3
+FOR /F %%I IN ('where python') DO (
+	CMD /C %%I -c "import sys; sys.exit(1 if not sys.version_info[:2] >= (3, 4) else 0)"
```

Comment:
Why Python 3.4 in particular? I'd imagine we could do just Python v3, no?

Reply:
that's what was there before. I'd imagine there was a good reason for choosing that